runtime/src/lib.rs

```diff
@@ -264,32 +264,34 @@ impl pallet_transaction_payment::Trait for Runtime {
 
 parameter_types! {
 	pub const TombstoneDeposit: Balance = 16 * MILLICENTS;
-	pub const RentByteFee: Balance = 100_000;
+	pub const RentByteFee: Balance = 4 * MILLICENTS;
```

Comment:
Was this an intended change?

Reply:
Ah I copied that from `node-runtime` by mistake - will revert to keep our local value